### ISSUE TITLE
Allow the option to choose the path to install to for external harddrives

### ISSUE THREAD

**Issue by jeffdupont** [marzent/XIV-on-Mac#99]
Is it possible to select the path in which the files would be downloaded to? I don't have enough room on my laptop to fit everything to download and move after the fact.

**Comment by seathasky** [comment]
I agree this would be a great feature. Maybe something we can look into doing soon @marzent

**Comment by marzent** [comment]
Currently you can already choose the path to install to after the initial wizard by selecting 
File -> Choose Game Path
in the global menu.

But yeah, would be nicer for the start dialog to also offer that option.

**Comment by CovertDuck** [comment]
If we were to include the option to move the install post-install, perhaps using symbolic links pointing to the new location would be a safer bet?

**Comment by marzent** [comment]
Using symbolic links there will be much more painful than what we currently have, especially if there are broken ones…

Both XL and XOM have a perfectly good concept of a game path that is also part of the launcher log (in fact that was one of the major breaking changes going from 1.X to 2.X, a non fixed location to point it to for the game files)

Also there are edge cases like what to do when a folder with an already existing game install is at the location the symbolic link is supposed to be created.

The copy option should probably just go, it was from the early beta where it was not clear the launcher would “damage” an existing install. The third option could just end up as “Download to a custom location” or something of that sort and would solve this issue.
